app/system/survey/js/databaseUtils.js

```diff
@@ -563,6 +563,12 @@ return {
                 // convert from a nanosecond-extended iso8601-style LOCAL TIME ZONE time HH:MM:SS.sssssssss
                 // this does not preserve the nanosecond field...
                 return odkCommon.toDateFromOdkTime(new Date(), value);
+            } else if ( jsonType.elementType === 'date_year_only' ) {
+                return moment(value).year();
+            } else if ( jsonType.elementType === 'date_month_only' ) {
+                return moment(value).month();

```

Comment:
I think this one is more important than year to use format

```
> moment('2000-07-01').format('MM')
"07"

> moment('2000-07-01').month()
6
```

So it's confusing both because it doesn't pad to 2 digits automatically and also appears to 0-index years so that January is 0 and July is 6